app/assets/javascripts/application/tools/congress_message.js

```diff
@@ -1,4 +1,50 @@
 $(document).on("ready", function() {
+  $(".congress-message-rep-lookup, #congress-message-create").on("ajax:beforeSend", function() {
+    submit_in_progress($(this));
+  });
+
+  $(".congress-message-rep-lookup").on("ajax:complete", function(xhr, data, status) {
+    $form = $(this);
```

Comment:
`$form`/`$phone` are given global scope but it looks like they're only needed within this file.